Sources/BraintreeSEPADebit/BTSEPADebitDriver.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+@objc public class BTSEPADebitDriver: NSObject {
```

Comment:
Random early two cents: we were talking about moving away from driver in iOS toward client at some point.